### PR TITLE
M5943: As BBMRI-ERIC I want the free text search box on top replaced with a search box in the facet that searches specific fields

### LINKED ISSUE #33: Search string does not get escaped properly
### How to Reproduce
Open the console.
In the search field, type "Wageningen University".

### Expected behavior
No error in console

### Observed behavior
Collections query causes a 400 error (bad request)

### PR DESCRIPTION
**Pull request by fdlk**
Depends on new release of rsql library.
Fixes #33: Search string does not get escaped properly

#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] User documentation updated
- [x] Clean commits
- [x] No warnings during install
- [x] Updated flow typing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "@molgenis/molgenis-api-client": "^3.0.0",
     "@molgenis/molgenis-i18n-js": "^0.0.5",
+    "@molgenis/rsql": "^0.1.0",
```
Comment (fdlk) [inline package.json]: Needs version 0.2.0

**Comment by marikaris** [comment]
A few findings:

1. The search term is not added to the selected filters, like the other filters. Therefore it is not clear if the filter is applied. And if I want to clear the filter I should clear the text field (for which is no button). Example:
screen shot 2018-08-22 at 15 02 50
I only get: country: germany and not "Hannover"

2. Shouldn't we have a clear button for the search field?

3. Some weird behavior:
When I search for "Hannover" in free text field and then select Austria, first I get "No biobanks..." and after a few seconds I get a result (which is weird because Hannover is not located in Austria):

screen shot 2018-08-22 at 14 54 29

When I first search on "Hannover", press enter and then apply the filter on Austria, I get a correct result:
screen shot 2018-08-22 at 14 55 22

**Comment by fdlk** [comment]
Search term is not selected in the filters => This is existing issue, #13

**Comment by fdlk** [comment]
Hannover / Austria => I think this is existing issue #28

**Pull request merged**